content-based-router/content-based-router.bal

```diff
@@ -0,0 +1,52 @@
+import ballerina/http;
+
+type UKresponse record {|
```

Comment:
```suggestion
type UKResponse record {|
```
??

Reply:
1) Formatting should be `UkResponse`
2) Better to mention the api, eg: `DhlResponse`

Reply:
Shall I change to `DhlUkResponse` and `DhlDpiResponse` since the endpoint responses are different or use a more generic `DhlResponse1`, `DhlResponse2`?